gravityflow/gravityflow
# Update edd sl class

This PR update the EDD SL Class to the latest version and adds some fixes. The EDD item name is now deprecated in  favour of the item ID. It also adds admin notices for invalid extension licenses, fixes admin notices on multisite and ensures that the admin notices are not displayed while the installation wizard is being displayed on single site.

**Testing instructions**
- Ensure the admin notices for Gravity Flow and extensions are only displayed on the main site.
- Ensure there are no regression issues for single site.
- Ensure the admin notices are not displayed while the installation wizard is being displayed.
